.gitignore

```diff
@@ -19,3 +19,4 @@ logs/
 miniircd/
 nums_basepoints.txt
 schedulefortesting
+scripts/commitmentlist
```

Comment:
@chris-belcher @AdamISZ ping - added this file to `.gitignore` so that transaction IDs aren't incidentally committed into VCS (git).

Reply:
Those aren't transaction ids, they're podle commitments (which are broadcast in public), but that aside, thanks, that's a good one to add :)